docker/docker-compose.prod.yml

```diff
@@ -91,7 +90,7 @@ services:
       POSTGRES_USER: ${DB_USERNAME}
       POSTGRES_DB: ${DB_DATABASE_NAME}
     volumes:
-      - pgdata:/var/lib/postgresql/data
+      - ${UPLOAD_LOCATION}/postgres:/var/lib/postgresql/data
```

Comment:
This doesn't work with NTFS drives. Postgres errors with `docker initdb: error: could not change permissions of directory "/var/lib/postgresql/data": Operation not permitted`.

Reply:
Do you actively use this file, but not the dev version?

Reply:
I use both, but I have separate copies of them so I can change hwaccel settings without having them show up in the diff.

Reply:
So, do you care if this change is made, since you use your own copies anyways?

Reply:
I'm fine with it personally. I think it can cause confusion if someone doesn't know about this quirk, though, so it'd be good to add a comment about it.

Reply:
I think [this](https://forums.docker.com/t/data-directory-var-lib-postgresql-data-pgdata-has-wrong-ownership/17963/24) comment gives the clearest explanation on why the error happens.

Reply:
Hmmk OK. This would be a current problem with the docker-compose.dev.yml file. Where do you think we should add a note for this situation?

Reply:
Maybe a blurb about it in the developer setup page and a generic comment in the docker-compose.dev.yml about checking this page for setup and troubleshooting.